lib/rutema/core/engine.rb

```diff
@@ -169,12 +270,25 @@ def initialize queue,configuration
       @streaming_reporters<<@collector
       @configuration=configuration
     end
-    #Call this to establish a queue with the given identifier
+
+    ##
+    # Call this to establish a queue with the given identifier
+    #
+    # This method will create a new queue within the Dispatcher instance into
+    # which data from the incoming queue from the Engine instance will be
+    # demultiplexed too.
+    #
+    # * +identifier+ - a unique identifier for the queue. If two identifiers
+    #   collide the later added subscriber will take the place of the earlier
+    #   one
     def subscribe identifier
       @queues[identifier]=Queue.new
       return @queues[identifier]
     end
-    
+
+    ##
+    # Start #update threads of all event/streaming reporters and then start own
```

Comment:
and then start a new locally managed thread

Reply:
Applied your suggestion :+1: